src/@types/index.d.ts

```diff
@@ -0,0 +1,193 @@
+interface FetchResponse {
+    json(): Promise<any>;
+}
+
+interface QueryParameters {
+    [key: string]: string;
+}
+
+//  Classifiers
+// ---------------------------------------------------------------------------
+type CountryCode = string;
+type ASN = number;
+type ConnectionType = string;
+type DeviceType = string;
+
+//  Server response
+// ---------------------------------------------------------------------------
+
+interface Test {
+    id: string;
+}
+
+interface Agent {
+    hasFeatureSupport: boolean;
+}
+
+interface Client {
+    country_code: CountryCode;
+    connection_type: ConnectionType;
+    asn: ASN;
+    device_type: DeviceType;
+    [key: string]: string | number;
+}
+
+interface Fastly {
```

Comment:
Is this the interface for the performance observer or is this an artifact of the fastly provider?

Reply:
Good catch. Actually it appears to be unused so I just removed it.